far/stddlg.cpp

```diff
@@ -258,6 +259,16 @@ int GetSearchReplaceString(
 	{
 		switch (Msg)
 		{
+		case DN_DRAWDIALOGDONE:
+			{
+				if (FirstKey)
+				{
+					Dlg->ProcessKey(*FirstKey);
```

Comment:
This used to be
https://github.com/FarGroup/FarManager/blob/833c93989cc63b3fde1a9f85552bd4d730db2b7b/far/viewer.cpp#L2594

Does it make any sense? Do I need these complications?